packages/calcite-components/package.json

```diff
@@ -19,7 +19,7 @@
   ],
   "scripts": {
     "build": "npm run util:prep-build-reqs && stencil build",
-    "postbuild": "npm run util:patch && git restore src/components/*/readme.md",
+    "postbuild": "npm run util:patch && npm run util:generate-t9n-docs-json && git restore src/components/*/readme.md || true",
```

Comment:
Is `util:generate-t9n-docs-json` throwing unnecessarily or is there another reason for the fallback `true`?

Reply:
Woops that was left over from my local testing. The docker container I use isn't a git repo so `git restore` fails. This is my workaround but I forgot to change it back, fixing!